web/pages/program/[slug].tsx

```diff
@@ -148,11 +148,32 @@ const SessionRoute = ({
     currentSessionInProgram?._id,
     sessions
   );
+  const hasAssociatedProgram = startDateTime && currentSessionInProgram;
   const hasHighlightedSpeakers =
     speakers?.length === 1 || speakers?.length === 2;
+  const formattedStartDate = formatDateWithDay(start, mainVenueTimezone, ', ');
+  const formattedTimezone = getNonLocationTimezone(
+    start,
+    mainVenueTimezone,
+    true
+  );
+  const formattedDuration =
+    hasAssociatedProgram &&
+    formatTimeRange(
+      start,
+      currentSessionInProgram?.duration,
```

Comment:
Nit: Why the added safe navigation? The preceding `hasAssociatedProgram &&` implies that `currentSessionInProgram` cannot be nullish.

Reply:
2f0880a.